evcouplings/complex/protocol.py

```diff
@@ -167,20 +167,21 @@ def describe_concatenation(annotation_file_1, annotation_file_2,
     )
     
     # If the user provided genome location files, calculate the number
-    # of ids for which we found an embl CDS
+    # of ids for which we found an embl CDS. Default value is np.nan
+    embl_cds1 = np.nan
+    embl_cds2 = np.nan
+
     if genome_location_filename_1 is not None and \
```

Comment:
My obligatory stylistic nitpicking (I think this is actually somewhere in PEP8) would be to wrap this with brackets and indent one level more: 
```python
if (genome_location_filename_1 is not None and 
        genome_location_filename_2 is not None):
```